web/modules/webspark/webspark_blocks/webspark_blocks.install

```diff
@@ -241,6 +241,20 @@ function webspark_blocks_update_9024(&$sandbox) {
   _webspark_blocks_revert_module_config();
 }
 
+/**
+ * WS2-1711 - Add new paragraph type for Image based card component.
+ */
+function webspark_blocks_update_9025(&$sandbox) {
+  _webspark_blocks_revert_module_config();
+}
+
+/**
+ * WS2-1711 - Add new paragraph type for Image based card component.
+ */
+function webspark_blocks_update_9026(&$sandbox) {
+  _webspark_blocks_revert_module_config();
+}
+
```

Comment:
@juanmitriatti You'll need to increment the update numbers on these since other update hooks have claimed these numbers. Also looks like there's a duplicate update hook. You only need one.